menus/pause_menu.gd

```diff
@@ -1,11 +1,28 @@
 extends Control
 
+var game_paused : bool = false:
+	get:
+		return game_paused
+	set(value):
+		game_paused = value
+		get_tree().paused = game_paused
+		_update_visibility()
```

Comment:
Why this function? It's only been used here 😅

Reply:
To improve readability since i was using if and else

Reply:
Done